### PR TITLE
Migration for new ext_lti_assignment_id column

### PR DESCRIPTION
**Pull request by marcospri**
For #2527

Generated from model changes in https://github.com/hypothesis/lms/pull/3118

Test migrating forward and backwards:

- `hdev alembic upgrade head`
- `hdev alembic downgrade d9c9e65c463e`

and check the output of `\d module_item_configurations` on `make sql`'s prompt

### CODE REVIEW

File: lms/migrations/versions/a58605fdd398_add_ext_lti_assignment_id.py
Comment (marcospri) [inline lms/migrations/versions/a58605fdd398_add_ext_lti_assignment_id.py:25]: Added by alermbic autogenerate. I guess it gives hints to generate the SQL.

**Review by seanh — APPROVED** [review]
This looks good but I think let's hold off on merging it until the other related PRs are approved, just in case we realise we need to change something

**Pull request merged**
